autoload/dirvish.vim

```diff
@@ -83,6 +83,7 @@ function! dirvish#shdo(l1, l2, cmd)
   write
   if executable('chmod')
     call system('chmod u+x '.tmpfile)
+    edit
```

Comment:
Why?
I really try to avoid using silent and noautocmd (I have seen some issue with regard to vim-diminactive not undimming when using dirvish - probably due to some usage of `noautocmd`)..

This is what vim-eunuch is doing: https://github.com/tpope/vim-eunuch/blob/5ee2b82b565e6c6d80f1cb7735c78f66a159b198/plugin/eunuch.vim#L200-L201.